### PR TITLE
Add k8s

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Give the k8s cluster api its own endpoint to make it easier to access directly (ex: for heartbeat)

### CODE REVIEW

**Review by patheard — COMMENTED** [review]
This looks good to me!  Before we merge it, I'd like to get the Staging DNS record added first just to make sure everything works as expected.

Let me know when you've got a few mins this morning and I can help make that happen.

**Comment by jimleroyer** [comment]
LGTM! but I'll let Pat follow up on the rest. If we need a merge and Pat was kidnapped though, ping me for the approval.

**Review by patheard — APPROVED** [review]
cds-snc/notification-terraform#339 will have to merge before the record works as expected, but you can go ahead and merge this PR ahead of time if you'd like since it'll just result in a redirect to the Notify homepage.

**Pull request merged**
